app/_hub/kong-inc/openid-connect/how-to/_demonstrating_proof_of_possession.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Demonstrating Proof-of-Possession
+nav_title: Demonstrating Proof-of-Possession
+minimum_version: 3.7.x
+---
+
+## Overview
+
+Demonstrating Proof-of-Possession(DPoP) is an alternative technique to the Mutual TLS certificate-bound access tokens. Instead of binding the token with the client certificate used in the mTLS, it binds the token with a JWK provided by the client.
+
```

Comment:
Add diagram here. Proposed draft:

```
<!--vale off-->
{% mermaid %}
sequenceDiagram
    autonumber
    participant client as Client <br>(e.g. mobile app)
    participant kong as API Gateway <br>(Kong)
    participant httpbin as Upstream <br>(backend service,<br> e.g. httpbin)
    activate client
    client->>client: generate key pair
    activate kong
    client->>kong: POST /oauth2/token<br>DPoP:$PROOF
    kong-->>client: DPoP bound access token ($AT)
    deactivate kong
    activate httpbin
    client->>httpbin: GET https://example.com/resource<br>Authorization: DPoP $AT<br>DPoP: $PROOF
    deactivate client
    httpbin->>httpbin: validate $AT and $PROOF
    activate client
    httpbin-->>client: response
    deactivate client
    deactivate httpbin
{% endmermaid %}
<!--vale on-->
```

Will render as something like this:

![Screenshot 2024-05-02 at 10 11 05 PM](https://github.com/Kong/docs.konghq.com/assets/54370747/746c671f-affc-45b5-b7d7-417933e7e3b7)